### ISSUE TITLE
Since Update to LXQT 0.12, icons are missing from taskbar

### ISSUE THREAD

**Issue by FrederikLauber** [lxde/lxqt#1362]
Hello,

since the new release of LXQT 0.12, I cannot see icons of programs running in the background anymore.
Examples are slack, nm-applet, pidgin.
They are still running in the background after they are started via autostart, are displayed for some while and then suddenly vanish to never return again.

[EDIT]

##### Expected Behavior
Icons should show  up

##### Current Behavior
Icons show up for a short while and then vanish

##### Possible Solution
Show the Icons as in lxqt 0.11

##### Steps to Reproduce (for bugs)
Put {pidgin, slack, nm-applet} into the session austostart
Restart your session
wait roughly 10 minutes

##### Context
This bug affects me because i cannot select wifis anymore on my laptop nor chat because pidgin and slack are not shown

##### System Information
* Distribution & Version: Arch Linux
* Kernel: 4.13.8-1-ARCH
* Qt Version: 5.9.2-1
* liblxqt Version:  0.12.0-1
* Package version: 0.12.0-1

**Comment by palinek** [comment]
@FrederikLauber why did you refuse to use the template for issues reporting? Please, use it... it is very frustrating to ask the same questions (distro, version etc...) for every reported issue.

**Comment by palinek** [comment]
@FrederikLauber (after the icons are vanished) what happens if you remove the "System tray" plugin and re-add it again ?

**Comment by FrederikLauber** [comment]
Hello,

while trying to solve an issue with the world clock, I accidentally must have solved this problem.

There was actually an error in my description of the problem. Pidgin and nm-applet got killed but slack keeps on running in the background so I think @palinek was already on the right track here that most likely the corresponding plugin crashed, taking the applications with it instead of not displaying the icons.

At the moment, I cannot reproduce the problem anymore. There are two things I changed, 1. run pacman -Syu a few times (normal system maintenance) and 2. delete all the old lxqt config (what was due to trying to fix a bug with the world clock). Now, the icons are displayed all the time.
